.gitignore

```diff
@@ -5,7 +5,8 @@
 #   git config --global core.excludesfile '~/.gitignore_global'
 
 #Ignore sandimeter
-/sandi_meter/
+*.sandi_meter
```

Comment:
Can you also remove all added sandi_meter files from the previous commits? 😄

Reply:
They aren't really needed

Reply:
Done! :D